packages/paypal-commerce-integration/src/paypal-commerce-types.ts

```diff
@@ -337,6 +337,21 @@ export interface ApproveCallbackActions {
     order: {
         get: () => Promise<PayPalOrderDetails>;
     };
+    restart(): void;
+}
+
+export interface ProviderError extends Error {
+    errors?: ErrorElement[];
+    status?: string;
+}
+
+interface ErrorElement {
+    code: string;
+    message: string;
+    // eslint-disable-next-line @typescript-eslint/naming-convention
+    provider_error?: {
+        code: string;
```

Comment:
Do we now full list of code value?

Reply:
Please explain) I didn't quit get it))

Reply:
If we have a list of values, for example `APPROVE`, `ERROR`, `PENDING` we can set it instead of `string`.
Because in the future it will be more understandable to the developers what possible value we can get

Reply:
No, we don't know what could be in code, it could be any string depends on what we send from BE